api/test/wfl/tools/fixtures.clj

```diff
@@ -110,6 +92,49 @@
     (create-local-database name)
     (setup-local-database name)))
 
+(defmacro with-fixtures
```

Comment:
Not sure if this should be a macro or a fixture.
If I made it a function and the consuming lambda unary, it would itself be a fixture which might be kinda nice.

Reply:
Guess you can do this as a macro? 
```clojure
(fixtures/with-fixtures
     [(fixtures/with-fixtures 
        [(fixtures/with-fixtures 
           [(fixtures/with-fixtures [])])])]
    identity)
=> [[[[]]]]
```